pallets/parachain-staking/src/mock.rs

```diff
@@ -27,17 +27,17 @@ use frame_support::{
 use sp_core::H256;
 use sp_io;
 use sp_runtime::{
-    testing::Header,
     traits::{BlakeTwo256, IdentityLookup},
     Perbill, Percent,
 };
 
 pub type AccountId = u64;
 pub type Balance = u128;
-pub type BlockNumber = u64;
 
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
 type Block = frame_system::mocking::MockBlock<Test>;
+type BlockNumber = u32;
+type Header = sp_runtime::generic::Header<BlockNumber, BlakeTwo256>;
```

Comment:
Can these types like `Header/BlockNumber` be imported from `primitives/manta/src/types.rs`?